hooks/hookexecution/executor.go

```diff
@@ -160,7 +160,7 @@ func (e *hookExecutor) ExecuteProcessedAuctionStage(request *openrtb_ext.Request
 
 	stageName := hooks.StageProcessedAuctionRequest.String()
 	executionCtx := e.newContext(stageName)
-	payload := hookstage.ProcessedAuctionRequestPayload{BidRequest: request.BidRequest}
+	payload := hookstage.ProcessedAuctionRequestPayload{BidRequest: request.BidRequest, BidRequestWrapper: request}
```

Comment:
Why do you need to pass both the Request and the Wrapper?

Reply:
to avoid changing codes for other unrelated modules(openrtb.blocking)